spec/controllers/orgs/rosters_controller_spec.rb

```diff
@@ -135,8 +117,7 @@
           before do
             post :create, params: {
               id: organization.slug,
-              identifiers: "    \r\n ",
-              identifier_name: "emails"
```

Comment:
There are a bunch of changes here to identifier_name stuff, didn't this happen in a previous PR? If not, can we move to a different PR? It's unrelated to the Google Classroom stuff and this PR is already massive.